courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -822,26 +815,27 @@ Ada.Characters.Handling Sample Content
 Language-Defined Type Boolean
 -------------------------------
 
-* An enumeration type
+* Enumeration
 
    .. code:: Ada
 
       type Boolean is ( False, True );
  
-* Supports assignment, relationals, attributes
+* Supports assignment, relational operators
+* Special attributes
```

Comment:
This bullet doesn't make sense for the code block. leave the code block under the previous bullet
The (original) "attributes" meant it supports all the normal tick-properties